src/Module.zig

```diff
@@ -383,6 +383,12 @@ pub const Decl = struct {
     /// For anonymous decls and also the root Decl for a File, this is 0.
     zir_decl_index: Zir.Inst.Index,
 
+    /// Whether the extern decl has a lib name populated.
+    /// Populated when tag is `extern_fn` or `extern`.
+    has_lib_name: bool,
```

Comment:
This does not need to exist because a `0` value for a `Zir.Inst.Index` can represent absence. As this is written we would be paying 4 bytes for 1 bit of data, per Decl, of which there are many.

However there is a bigger issue with this patch; see below...